safari/Forem for Safari/Info.plist

```diff
@@ -17,7 +17,7 @@
 	<key>CFBundlePackageType</key>
 	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.1.3</string>
+	<string>0.1.4</string>
```

Comment:
**Non-blocking question/suggestion:** Given that our APIs (`/valid_forems.json` and `discover.forem.com/api/forems/listed` are responding with `"meta":{"latestExtensionVersion":"0.2.0"}` would it make sense to bump all of our extensions to `0.2.0` or go with `0.2.1` and start fresh from there?

This is just me wondering if keeping them all in sync makes sense, but I don't think it would make a major difference overall because these aren't affecting how the extension works internally AFAICT.

Reply:
I...don't know that we want to tie the version of the browser extension to the version of the discover API: they are different products! That said: I did rev a bunch of the `0.1.3` entries in `manifest.json` to `0.1.4`.

Reply:
Okay, this is what _I think I understand_ (sharing it just to be sure it makes sense):

The version `0.2.0` that is currently available in both [/valid_forems.json](https://forem.com/valid_forems.json) and [discover.forem.com/api/forems/listed](https://discover.forem.com/api/forems/listed) payloads is just a "flag".

When a new version of the browser extension is shipped and we want users to upgrade, we can just bump it both in the source code and the JSON endpoints. That way, if the JSON payload doesn't match the source code of the browser extension it means the code is outdated and the user should be suggested to upgrade.

The above ^ reasoning is based on the browser extension [logic in place](https://github.com/forem/forem-browser-extension/blob/main/content/content.js#L50-L60) and the [first commit it was added](https://github.com/forem/forem-browser-extension/commit/1e2a57f2a67e3a9cfe2f4cca7bf001cc702192ae#diff-9b804a9bbecde0819c0b17d54055d09095a0f0331a17ad4c24ba6a2c0110c236R18).

When I first saw this I thought it made sense to keep this version and the actual browser extension versions in sync, mostly because of the name `"latestExtensionVersion"` (also they're on _similar version numbers_ so I thought it was a typo/human error they mismatched). But in reality, this could just as easily be an integer number that we can be manually increase anytime we want to suggest the user should upgrade their browser extension.

If all of this rambling makes sense, then you are right and we have no need to change anything here (no real need to tie it with the actual store version numbers) 👍

Reply:
Oh yeah, that pesky `latestExtensionVersion` field! Yeah: let's keep it as is for now, but I can definitely see yanking that at some point: seems like a bit of over engineering right now!